.npmrc

```diff
@@ -1 +1 @@
-legacy-peer-deps=true
+engine-strict=true
```

Comment:
Not a blocker, but just a note... I'm still not 100% sure what this does after reading the [docs](https://docs.npmjs.com/cli/v8/using-npm/config#engine-strict):

_If set to true, then npm will stubbornly refuse to install (or even consider installing) any package that claims to not be compatible with the current Node.js version._

So, I guess this checks _packages that will be installed_ to make sure they're compatible with the given NPM version range specified in `package.json`? Because, at first glance, I assumed this would enforce that the version of NPM running fits within a certain version range. 🤕

Reply:
If I understand correctly, strict mode will cause `npm install` time fail with an error instead of a warning.  

I actually saw this in the CI build before I updated the `setup-node` job to the correct version.

I think this makes the engine mismatch more explicit and easy to troubleshoot, as opposed to a warning that may get ignored and cause unforeseen problems.